manifests/backends/ldap.pp

```diff
@@ -1,11 +1,12 @@
 # ldap backend for powerdns
-class powerdns::backends::ldap ($package_ensure = $powerdns::params::default_package_ensure) inherits powerdns {
+class powerdns::backends::ldap (
```

Comment:
Is it the convention to add the `inherits powerdns` on a new line?

Reply:
It's not explicitly written, but if you read through https://www.puppet.com/docs/puppet/8/style_guide.html#style_guide_classes-internal-organization-classes-defined-types it seems logical. Besides, if there are multiple parameters, it's easier to read this way.

That's why I tried (if I've not kissed anything) to stick to the same schema.